### PR TITLE
minor: Add deploy with balena button to README

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

File: balena.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+assets:
+  repository:
+    type: blob.asset
+    data:
+      url: 'https://github.com/balena-os/leviathan'
+data:
+  applicationEnvironmentVariables:
+    - TESTBOT_DUT_TYPE=
+    - SCREEN_CAPTURE=
+    - TESTBOT_PSU_ADDRESS=
```
Comment (klutchell) [inline balena.yml:15]: I'm not sure blank fields are allowed here, does this actually deploy?

**Comment by klutchell** [comment]
@balena-ci rebase

File: balena.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  repository:
+    type: blob.asset
+    data:
+      url: 'https://github.com/balena-os/leviathan'
+data:
+  applicationEnvironmentVariables:
+    - TESTBOT_DUT_TYPE:
+    - SCREEN_CAPTURE:
+    - TESTBOT_PSU_ADDRESS:
+    - BALENA_HOST_CONFIG_dtoverlay: "balena-fin","uart1,txd1_pin=32,rxd1_pin=33"
```
Comment (klutchell) [inline balena.yml:16]:
```suggestion
    - BALENA_HOST_CONFIG_dtoverlay: >
      "balena-fin","uart1,txd1_pin=32,rxd1_pin=33"
```
I think a scalar may avoid the need to escape the double quotes. There are other workarounds as well.

**Comment by klutchell** [comment]
I fixed up the formatting of the balena.yml so we should be good to go. I would approve but you added the draft label so I leave it to you!

**Comment by vipulgupta2048** [comment]
Yeah, I wanted to add some sane defaults or prompts for the user. Added persistent logging as well.

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**